transplant_guide.md

```diff
@@ -147,39 +147,67 @@ Info about U-district
+**[ORCA LIFT](http://metro.kingcounty.gov/programs-projects/orca-lift/)**
+  So… you’re poor! Since it’s likely you’ll have zero income once attending Ada, it’s nice to know there are some perks with falling below the poverty line. The first of few is reduced bus fare! To apply for a reduced fare bus pass (the ORCA LIFT card), you’ll need to stop by one of these enrollment offices and provide proof of your eligibility and your identity.
+  We’ll warn you now: there are tons of locations, with tons of different hours. There's [one near Ada](http://www.kingcounty.gov/healthservices/health/locations/downtown.aspx) that's open 8am - 5pm. If you have your paperwork ready and show up right when they open (so there's little chance of a line), you can easily be out in 10-15 minutes.
+  They make you prove your income, and they have [tons of different options](http://metro.kingcounty.gov/programs-projects/orca-lift/to-qualify.html) for how to do that. Read carefully, and show up prepared. If you get Apple Health before you get the ORCA LIFT pass, they'll have it on file and you won't need to prove your income again.
+  Note: If you don't have any qualifying documentation to show your income, you can submit a request for your employment history (which can be done at the enrollment office). Expect it to take about a week, but you can ask the enrollment office to mail your ORCA LIFT card to your home so you don’t have to find another time to visit during their limited hours.
+  Note2: If you’re applying for the free Apple Care healthcare (which you can do at some of the ORCA LIFT enrollment offices), you’ll be able to skip this step since your lack of income will be verified then.
 
-- biking
-  - tips/tricks?
+**Bus Apps**
+  There are a lot of apps that can tell you which buses to take to get home. Options include:
+  - Google Maps
+  - SOMEONE ELSE NEEDS TO EXPAND THIS! I just use Google Maps :P
+
+**Biking**
+  If you're new to Seattle, here's a warning: the hills are no joke. The thing about the hills is that there aren't a bunch of little hills.... it's that Seattle itself can seem like one GIANT hill. Depending on where you live, it can be a good 30 minute ride uphill the entire way. (It makes getting _to_ Ada easy, though!)
+  That being said, it's not a huge deal as long as you don't have a fixed-gear bike, and as long as you shift down. Don't sprint up the hills -- take your time. And don't feel silly if you get off and walk for a particularly steep block or two -- lots of people do it.
+  See our car-to-bike guide for more information!
+
+**The Ferry**
+  We _strongly_ recommend not living far enough away that you need to take the ferry, simply because commuting takes so much time. That being said, the ferry is a smooth ride, so if you're wondering if you can work while you're on the ferry, the answer is yes.
+
+**The Link**
+  - SOMEONE ELSE?
```

Comment:
Yeah, I'll try to get to it tonight or tomorrow. I can add a few more details to the ferry section, too, having commuted for several years from Vashon to Seattle & Tacoma by all three available ferry routes.